### DISCUSSION TITLE
DevGateway file is not available in the latest Git code.

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#125] [Q&A]
Hi Team, 

When attempting to run the Microsoft Fabric workload development sample Git code in local dev environment, we are unable to create any workloads because the DevGateway folder is missing from the backend files, unlike in previous commits. Is there a specific reason why the DevGateway file was removed from the Git repository? Also, is there any other way to create a workload in the local environment without the DevGateway?

Thank you

**Comment by AviSander** [comment]
hi. Please use the official public version of the DevGateway which can be found [here](url).https://www.microsoft.com/en-my/download/details.aspx?id=105993&msockid=2cd5f5fb29f6621c1f86fa5c288b6355
